scripts/seed-data

```diff
@@ -13,5 +13,5 @@ fi
 mkdir -p ${BASE_DIR}/lib/rancher/conf
 
 for yaml in "${CLOUD_DATA}" '/scripts/conf/rancher.yml';do 
-    cp $yaml ${BASE_DIR}/lib/rancher/conf/
+    cp $yaml ${BASE_DIR}/lib/rancher/conf/rancher.yml
```

Comment:
Regardless of the source file name, may sure it is always rancher.yml on the target